batch/compile_prenight_metadata_cache.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#SBATCH --account=rubin:developers      # Account name
+#SBATCH --job-name=auxtel_prenight_daily   # Job name
+#SBATCH --output=/sdf/data/rubin/shared/scheduler/prenight/sbatch/compile_prenight_metadata_cache.out # Output file (stdout)
+#SBATCH --error=/sdf/data/rubin/shared/scheduler/prenight/sbatch/compile_prenight_metadata_cache.err  # Error file (stderr)
+#SBATCH --partition=milano              # Partition (queue) names
+#SBATCH --nodes=1                       # Number of nodes
+#SBATCH --ntasks=1                      # Number of tasks run in parallel
+#SBATCH --cpus-per-task=1               # Number of CPUs per task
+#SBATCH --mem=16G                       # Requested memory
```

Comment:
I'm wondering about the configuration for this job. 
I think it's asking for 1 cpu, and the memory footprint for a single cpu on the milano nodes is generally pretty small (I think it's about 4GB). 
I wouldn't have expected this particular job to need 16GB, so if it doesn't, it seems like it would be easier for scheduling to have the requested memory be a bit smaller.